tests/foreman/api/test_syncplan.py

```diff
@@ -4,11 +4,16 @@
 from fauxfactory import gen_string
 from robottelo import entities
 from robottelo.common.decorators import (
-    data, run_only_on, stubbed)
+    data,
+    run_only_on,
+    skip_if_bug_open,
+    stubbed,
+)
 from robottelo.test import APITestCase
 # (too-many-public-methods) pylint:disable=R0904
 
 
+@stubbed
```

Comment:
Tests results when using `@stubbed()` on the classes:

    $ nosetests tests/foreman/api/test_syncplan.py
    SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS..S.SSSSSSSSSSSSSSSSSSSSSSSSS
    ----------------------------------------------------------------------
    Ran 59 tests in 17.663s

    OK (SKIP=56)

Test results when using `@stubbed` on the classes:

    $ nosetests tests/foreman/api/test_syncplan.py
    ..S.S
    ----------------------------------------------------------------------
    Ran 5 tests in 10.836s

    OK (SKIP=2)